### DISCUSSION TITLE
Minor problem with custom collection field UI

### DISCUSSION THREAD

**Discussion by johnrisby** [swellstores/community#372] [Help] [answered]
Hi,

I've added a custom field, a collection, and I have two minor issues. 

Firstly, one of the fields is a price so I'm using the number field and currency type. In the form to design/edit the field, it correctly shows with our base currency symbol (GBP) - first image.

But on the admin product page itself, where the field is displayed, the display uses $ which is not right and is going to be confusing for our staff - second image.

And the collection has 5 fields, but only the first 3 show up in the preview and you have to click Edit to view the rest of the details, which is not very intuitive. Again, I'm thinking about staff using the system. They shouldn't have to think about editing something just to view the details.

Is there any way around this?

Thanks
John
<img width="658" alt="CleanShot 2023-09-13 at 19 57 09@2x" src="https://github.com/swellstores/community/assets/1823505/1c40e85b-2a40-4596-b0c8-c77d4f155b35">
![CleanShot 2023-09-13 at 19 56 13@2x](https://github.com/swellstores/community/assets/1823505/bf7eca1e-8587-4d62-b774-3a5801ea8be4)

**Comment by logeshswell** [comment, marked as answer]
It looks like the currency issue that you reported is no present now. Can you check and confirm?

Regarding your second query about the display of fields, we restricted the table to show only the first 3 fields for neat UI.
